### PR TITLE
remove pdf package dependency

### PR DESCRIPTION
**Pull request by debermudez**
Remove the pdf package dependency if wkhtmltopdf isn't installed

Create html reports if pdf reports cannot be generated.

### CODE REVIEW

File: model_analyzer/reports/report_factory.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    PDF_PACKAGE = "wkhtmltopdf"
+
+    @staticmethod
+    def create_report():
+        if ReportFactory._is_apt_package_installed(
+                f"{ReportFactory.PDF_PACKAGE}"):
+            return ReportFactory.create_pdf_report()
+        else:
+            print(f"Warning: {ReportFactory.PDF_PACKAGE} is not installed. Pdf reports cannot be generated. "\
```
Comment (tgerdesnv) [inline model_analyzer/reports/report_factory.py:33]: print? Or one of our logging functions?

**Review by tgerdesnv — CHANGES_REQUESTED** [review]

File: model_analyzer/reports/report_factory.py (hunk 1 above)
Comment (debermudez) [reply model_analyzer/reports/report_factory.py:33]: I didn't think the logging streams were printed to the console. I am wrong.  I will update this.

Comment (debermudez) [reply model_analyzer/reports/report_factory.py]: Updated

File: model_analyzer/reports/pdf_report.py [hunk 2]
```diff
@@ -12,158 +12,22 @@
 # See the License for the specific language governing permissions and
```
Comment (nv-braf) [inline model_analyzer/reports/pdf_report.py:12]: Change the copyright date

File: model_analyzer/reports/report.py [hunk 3]
```diff
@@ -75,3 +75,10 @@ def write_report(self, filename):
         filename : str
             The name of the report
         """
+
```
Comment (nv-braf) [inline model_analyzer/reports/report.py:78]: Change the copyright date

File: tests/test_pdf_report.py [hunk 4]
```diff
@@ -31,101 +31,6 @@ def setUp(self):
         self.maxDiff = None
```
Comment (nv-braf) [inline tests/test_pdf_report.py:31]: Change copyright date

File: tests/test_report_manager.py [hunk 5]
```diff
@@ -109,7 +109,11 @@ def setUp(self):
             "model_analyzer.config.input.config_utils"
         ])
         self.os_mock.start()
+        # Required patch ordering here
```
Comment (nv-braf) [inline tests/test_report_manager.py:112]: Change copyright date

**Review by nv-braf — COMMENTED** [review]
Overall it looks good

File: model_analyzer/reports/report_factory.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+                f"{ReportFactory.PDF_PACKAGE}"):
+            return ReportFactory.create_pdf_report()
+        else:
+            logging.warning(f"Warning: {ReportFactory.PDF_PACKAGE} is not installed. "\
+                f"Pdf reports cannot be generated. Html reports will be generated instead."
+                )
+            return ReportFactory.create_html_report()
+
+    @staticmethod
+    def create_pdf_report():
```
Comment (tgerdesnv) [inline model_analyzer/reports/report_factory.py:43]: should this function (and create_html_report) start with an underscore? We don't want the user calling them directly, right?

File: model_analyzer/reports/report.py (hunk 3 above)
Comment (debermudez) [reply model_analyzer/reports/report.py:78]: Updated

File: tests/test_pdf_report.py (hunk 4 above)
Comment (debermudez) [reply tests/test_pdf_report.py:31]: Updated

File: tests/test_report_manager.py (hunk 5 above)
Comment (debermudez) [reply tests/test_report_manager.py:112]: Updated

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**